waspc/cli/Wasp/Cli/Command/Db/Migrate.hs

```diff
@@ -1,31 +1,25 @@
 module Wasp.Cli.Command.Db.Migrate
   ( migrateDev,
-    copyDbMigrationsDir,
-    MigrationDirCopyDirection (..),
   )
 where
 
-import Control.Monad.Catch (catch)
 import Control.Monad.Except (throwError)
 import Control.Monad.IO.Class (liftIO)
 -- Wasp generator interface.
 
-import qualified Path as P
-import qualified Path.IO as PathIO
-import StrongPath (Abs, Dir, Path', (</>))
-import qualified StrongPath.Path as SP.Path
+import StrongPath ((</>))
 import Wasp.Cli.Command (Command, CommandError (..))
 import Wasp.Cli.Command.Common
   ( findWaspProjectRootDirFromCwd,
     waspSaysC,
   )
 import qualified Wasp.Cli.Common as Cli.Common
 import Wasp.Cli.Terminal (asWaspFailureMessage, asWaspStartMessage, asWaspSuccessMessage)
-import Wasp.Common (WaspProjectDir, dbMigrationsDirInWaspProjectDir)
-import Wasp.Generator.Common (ProjectRootDir)
-import Wasp.Generator.DbGenerator (dbMigrationsDirInDbRootDir, dbRootDirInProjectRootDir)
 import qualified Wasp.Generator.DbGenerator.Operations as DbOps
 
+-- | Note(shayne): Migrates using migrations dir in project dir. The source down to project
```

Comment:
```suggestion
-- | NOTE(shayne): Migrates using migrations dir in project dir. The source down to project
```
I think we've been capitalizing TODO/NOTE etc, but not sure if that is a part of our style guide or we're flexible - @Martinsos ? #nitpicking :D

Reply:
Aha, good point -> I would vote for capitalizing them, as then it is easier to recognize they have special meaning. Btw for me, in Emacs, if I capitalize, they get colored differently, so that is also why I am doing it :D. But we don't have official style guide on this, no.

Reply:
Good point, fully agree capital is better, will update 👍🏻